lib/initializer.rb

```diff
@@ -0,0 +1,47 @@
+require_relative './game.rb'
+require_relative './board/board.rb'
+require_relative './board/board_checker.rb'
+require_relative './players/ai.rb'
+require_relative './players/human.rb'
+require_relative './players/player.rb'
+require_relative './ui/user_interface.rb'
+require_relative './ui/script.rb'
+
+def initializer(size, empty_char)
+  board = Board.new(size, empty_char)
+  checker = BoardChecker.new(size, empty_char)
+  script = Script.new
+  user_interface = UserInterface.new(board, script)
+
+  user_interface.welcome
+  mode_selection = user_interface.what_mode?
+
+  unless(mode_selection == 'computerVScomputer')
+    player_selection = user_interface.who_goes_first?
+  end
+
+  behavior1, behavior2, other = setup_behavior(mode_selection, 'X', 'O', board, checker, user_interface)
+  player1, player2, you = setup_players(player_selection, behavior1, behavior2, user_interface)
+
+  game = Game.new(board, checker, player1, player2, you, other)
```

Comment:
I think that the game initializer is great, it takes exactly what is needed for a game of tic-tac-toe, although the player1/player2/you/other feels a little redundant. Is there a way to merge the two concepts?